common/nacos_server/nacos_server.go

```diff
@@ -313,7 +305,7 @@ func injectSecurityInfo(server *NacosServer, param map[string]string) {
 }
 
 func getAddress(cfg constant.ServerConfig) string {
-	return cfg.IpAddr + ":" + strconv.Itoa(int(cfg.Port))
+	return cfg.Scheme + "://" + cfg.IpAddr + ":" + strconv.Itoa(int(cfg.Port))
```

Comment:
pls verify whether ipaddr starts with http:// or https://，some old user may config like this ‘http://console.nacos.io’

Reply:
ok, i will